src/Document.java

```diff
@@ -0,0 +1,23 @@
+public class Document {
+    private String title;
+    private String date;
+    private String author;
+
+    void setTitle(String title) {
```

Comment:
If you have private/protected fields, you should mandatory add BOTH getters and setters even if some of them are not needed now